cli/main.c

```diff
@@ -940,23 +962,19 @@ static int port_bind_info(int argc, char **argv)
 
 	ret = switchtec_bind_info(cfg.dev, &bind_status, cfg.phy_port);
 
-	if (ret < 0) {
-		printf("bind_info_error: %f", ret);
+	if (ret > 0) {
+		printf("bind_info_error: %d\n", ret);
```

Comment:
Probably should also be using switchtec_perror()

Reply:
Because same error codes were used by different MRPC commands and have different meaning. so  we just display the error code now.

Reply:
Well returning a number is incredibly user unfriendly. You should fix that and decode the number into a useful error message. If you can use switchtec_perror() that would be great, if not you should create something else. switchtec_bind_perror() or something.